packages/blade/src/components/Icons/AlertTriangleIcon/AlertTriangleIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ReactElement } from 'react';
+import { Svg, Path } from '../_Svg';
+import type { IconProps } from '..';
+import useIconProps from '../useIconProps';
+
+const AlertTriangleIcon = ({ size, color }: IconProps): ReactElement => {
+  const { height, width, iconColor } = useIconProps({ size, color });
+
+  return (
+    <Svg width={width} height={height} viewBox="0 0 20 20" fill="none">
```

Comment:
Why is your viewBox 20 20? Its 24 24 for every icon on figma. Are you copying the correct SVG? When I tried it, the results are quite different
<img width="394" alt="Screenshot 2022-08-26 at 11 44 50 AM" src="https://user-images.githubusercontent.com/24487274/186834898-99d44455-667e-4bbf-b8af-8c093a940b37.png">

Reply:
Thanks for the 🦅 👁️ review. Updated in https://github.com/razorpay/blade/pull/670/commits/569606161204170a8bc2f730c5e7a8f5dde139aa

Also tried out a SVGR config which works well for copy pasta. [Playground url](https://react-svgr.com/playground/?expandProps=none&native=true&replaceAttrValues=%23132644%3D%7BiconColor%7D&svgProps=width%3D%7Bwidth%7D%2Cheight%3D%7Bheight%7D&svgo=false)